apps/design/frontend/src/tabulation_screen.test.tsx

```diff
@@ -66,17 +66,17 @@ test('mark thresholds', async () => {
 
   userEvent.click(screen.getByRole('button', { name: 'Edit' }));
 
-  // Due to some weirdness with the tests, we can't clear the input before
-  // typing, so we have to just append
-  userEvent.type(definiteInput, '9');
-  userEvent.type(marginalInput, '8');
```

Comment:
This was actually causing invalid data to be entered (because `step={0.1}`) and so the form was not submitting in the tests. Now it properly prevents submission, so I had to change this to backspace the "7" before typing the "8".